test/unit/model/result/ResultServiceTest.php

```diff
@@ -197,6 +196,6 @@ public function testDeliveryExecutionNotFound()
         $this->assertNotSame($response->getMessageIdentifier(), $response->getMessageRefIdentifier());
         $this->assertSame('msg_identifier', $response->getMessageRefIdentifier());
         $this->assertSame('replaceResultRequest', $response->getOperationRefIdentifier());
-        $this->assertContains('de_id', $response->getStatusDescription());
+        $this->assertStringContainsString('de_id', $response->getStatusDescription());
```

Comment:
Why was this change necessary?

Reply:
It was suggested change.

Reply:
in the method `assertContains` we can find this warning:
```Using assertContains() with string haystacks is deprecated```

Reply:
thanks for the clarification :)